versions.js

```diff
@@ -42,4 +42,15 @@ const versionsPerRelease = new Map([
   ['73.0.3679.0', 'v1.12.2'],
 ]);
 
-module.exports = versionsPerRelease;
+const lastMaintainedChromiumVersion = '97.0.4692.0';
```

Comment:
Let’s add a comment here explaining which version this is supposed to refer to, i.e. the same major version as the current Chrome Stable per https://chromestatus.com/roadmap.

Reply:
Done, PTAL